sources/packages/api/src/constants/error-code.constants.ts

```diff
@@ -19,3 +19,24 @@ export const SIN_VALIDATION_RECORD_NOT_FOUND =
   "SIN_VALIDATION_RECORD_NOT_FOUND";
 export const SIN_VALIDATION_RECORD_INVALID_OPERATION =
   "SIN_VALIDATION_RECORD_INVALID_OPERATION";
+/**
+ * An user is trying to be added to the institution when it is already present.
+ */
+export const INSTITUTION_USER_ALREADY_EXISTS =
+  "INSTITUTION_USER_ALREADY_EXISTS";
+/**
+ * Only one legal signing authority is allowed per institution.
+ * If there is attempt to assign a second user as legal signing authority,
+ * a exception with this code will be thrown.
+ */
+export const LEGAL_SIGNING_AUTHORITY_EXIST = "LEGAL_SIGNING_AUTHORITY_EXIST";
+/**
+ * An institution must have at least on admin user always present. If the user
```

Comment:
I am trying to understand this scenario.

will the UI allow an admin user to change their own user type from admin to user? If no, these validations are for API I believe.

Reply:
Right now there is nothing preventing the user from doing that unless he is the last admin.